test/validation.test.js

```diff
@@ -86,6 +86,48 @@ describe("plugins/validation", function () {
                     expect(schema.validators.age[0].name).to.eql(validators.max().name);
                 });
 
+                it("should add a minLength validator", function () {
+
+                    var schema = new Schema({
+                        name: {
+                            type: String,
+                            minLength: 5
+                        }
+                    });
+
+                    expect(schema.validators.name).to.be.an("array");
```

Comment:
Just a small thing, but I think it should be: `expect(schema.validators.name).to.be("array");`, because `name` should not be *an* array, but it should be the string `"array"`. The test is ok though, it's just for the developer.